proxy/package.json

```diff
@@ -11,14 +11,14 @@
     "serve-frontends": "concurrently \"yarn serve-ui\" \"yarn serve-legacy\"",
     "serve-base": "concurrently \"yarn serve-root\" \"yarn serve-proxy\"",
     "serve-base-wait": "wait-on http://0.0.0.0:8401 && wait-on http://0.0.0.0:8402 && yarn serve-base",
-    "serve-proxy": "node ./index.js",
+    "serve-proxy": "nodemon ./index.js",
```

Comment:
This is a drive-by to aid in development so that the proxy is reloaded when the express files are changed. This also watches for changes to .env.local so you can now change MAASes without restarting the proxy! I'm not sure why we haven't done this before...